packages/design-system/src/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { Icon as IconComponent } from "./Icon";
```

Comment:
Why are we importing this as IconComponent? Let us rename the story using the `.storyname` method instead.

Reply:
.storyname?